src/system/game-speed.ts

```diff
@@ -21,7 +19,7 @@ export function initGameSpeed() {
     if (num instanceof FixedNumber) {
       return num.value;
     }
-    return thisArg.gameSpeed === 1 ? num : Math.ceil((num /= thisArg.gameSpeed));
+    return settings.settings.general.gameSpeed === 1 ? num : Math.ceil((num /= settings.settings.general.gameSpeed));
```

Comment:
```suggestion
    return settings.general.gameSpeed === 1 ? num : Math.ceil((num /= settings.general.gameSpeed));
```

why did that even work